hidden/Features/StatusBar/StatusBarController.swift

```diff
@@ -122,6 +152,18 @@ class StatusBarController{
     }
     
     @objc func openPreferenceViewControllerIfNeeded() {
-        Util.showPrefWindow()
+        let window = Util.showPrefWindow() as! PreferencesWindow
+        editPermHide()
+        window.windowClosedHandler {
+            self.exitEditPermHide()
+        }
```

Comment:
Memory leak by creating a retail cycle
 window.windowClosedHandler { [weak self] in
              self?.exitEditPermHide()
  }